core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -156,7 +172,7 @@ class QuotasTest extends KafkaServerTestHarness {
                                             RequestKeys.nameForKey(RequestKeys.ProduceKey),
                                             "Tracking throttle-time per client",
                                             "client-id", producerId2)
-    Assert.assertEquals("Should not have been throttled", 0.0, allMetrics(producerMetricName).value())
+    assertEquals("Should not have been throttled", 0.0, allMetrics(producerMetricName).value(), EPS)
```

Comment:
The assertEquals(expected, actual) method has been deprecated, so I used the assertEquals(expected, actual, delta) method. I can set the delta to zero in this case.